### PR TITLE
Update Buttons BorderThickness to 1px

### PR DESCRIPTION
**Pull request by savatia**
This commit updates all the Button controls borders to use a thickness of 1px (#835, #1144).

### CODE REVIEW

File: dev/SplitButton/SplitButton_rs1.xaml [hunk 1]
```diff
@@ -304,7 +303,7 @@
                                     FontSize="12"
                                     Text="&#xE70D;"
                                     VerticalAlignment="Center"
-                                    HorizontalAlignment="Right"
+                                    HorizontalAlignment="Center"
```
Comment (kaiguo) [inline dev/SplitButton/SplitButton_rs1.xaml]: Why changing the alignment?

Comment (savatia) [reply dev/SplitButton/SplitButton_rs1.xaml]: If I removed the right padding and instead centered it horizontally, if I leave it the way it is, it's not centered 
image.
The other approach is to adjust the padding, but this way is more dynamic.

Comment (mdtauk) [reply dev/SplitButton/SplitButton_rs1.xaml]: There is a design change coming to the Split Button #986 with a 1px gap between the Chevron and the Button, so having the text Centred will make sense for that.

image

**Pull request merged**